far/FarEng.hlf.m4

```diff
@@ -3364,47 +3364,56 @@ the contents of the file on the disk.
 the external program will be lost.
 
 @CodePagesMenu
-$ #Code pages menu#
-    This menu allows code page selection in the editor and viewer.
+$ #Code page selection menu#
+    This menu allows to select code page in the editor and viewer.
 
-    The menu is divided into several parts:
+    The menu is divided into several sections:
 
-    #Automatic detection# - Far tries to autodetect the code page of the text;
+        #Automatic# - Far will try to autodetect
+        #detection#   the code page of the text;
 
-    #System# - main 8-bit system code pages - ANSI and OEM;
+        #System#    - ^<wrap>main single-byte system code pages - ANSI
+and OEM;
 
-    #Unicode# - Unicode code pages;
+        #Unicode#   - ^<wrap>Unicode code pages;
 
-    #Favorites# - user controlled list of code pages;
+        #Favorites# - ^<wrap>code pages selected by the user;
 
-    #Other# - the rest of code pages installed in the system.
+        #Other#     - ^<wrap>the rest of code pages installed
+in the system.
 
-    The menu has two modes: full mode with visible #Other# section and brief
-mode with hidden #Other# section. The modes can be switched by pressing #Ctrl+H#.
+    The following key combinations are available in this menu:
 
-    Current menu mode state corresponds to ~far:config~@FarConfig@ parameter #Codepages.CPMenuMode#.
-It can alter the way of ~code page autodetect~@CodePageAuto@ in builtin Editor/Viewer.
-See also ~far:config Codepages.NoAutoDetectCP~@Codepages.NoAutoDetectCP@ parameter.
+        #Ctrl+H# - ^<wrap>Shows or hides the #Other# menu section.
+The current mode is stored in the #Codepages.CPMenuMode# parameter
+in ~far:config~@FarConfig@.
 
-    #Ins# keypress moves code page from #Other# to #Favorites#, #Del# moves the
-code page back. #F4# allows to rename #Favorites# and #Other# code pages (renamed code pages will be marked with the #*#).
+        #Ins#    - ^<wrap>Moves the code page from the #Other#
+section to the #Favorites# section.
 
-    Диалог ~Изменение имени кодовой страницы~@EditCodePageNameDlg@
+        #Del#    - ^<wrap>Moves the code page from the #Favorites#
+section back to the #Other# section.
 
-    See also: common ~menu~@MenuCmd@ keyboard commands.
+        #F4#     - ^<wrap>Opens the
+~Rename code page~@EditCodePageNameDlg@ dialog. Only #Favorites# and
+#Other# code pages can be renamed. The renamed code pages are indicated
+with the #*# symbol.
 
-@EditCodePageNameDlg
-$ #Изменение имени кодовой страницы#
-    Диалог #Изменение имени кодовой страницы# позволяет изменить отображаемое имя для
-#избранных# и #прочих# кодовых страниц.
+    The #Codepages.CPMenuMode# and
+~Codepages.NoAutoDetectCP~@Codepages.NoAutoDetectCP@ parameters in
+~far:config~@FarConfig@ affect ~code page autodetect~@CodePageAuto@
+algorithm in the editor and viewer.
```

Comment:
This sentence is somewhat confusing - ok, the parameters do affect the algorithm, but how? Would it detect more code pages? Or would it make less errors? Should I be worried about it?

If it's problematic to describe the actual impact it would be probably better to skip this at all (for now).

(same for FarRus.hlf.m4)

Reply:
Well, I tried to faithfully retain all the information from the original. I agree that the phrase is not very useful. I guess the main goal was to mention the `far:config` parameters affecting code page auto-detection. Actually, this piece belongs to the `@CodePageAuto` help page. And that page is pretty useless as well.

Let me remove this sentence here and create a task to improve the `@CodePageAuto` page. I'll try to describe how the parameters affect the algorithm.